src/app/background/sagas/watchActivatedTab.saga.ts

```diff
@@ -27,7 +27,7 @@ export default function*() {
 
       if (!tab || !tab.url) {
         yield call(resetBadge, tabId);
-        yield call(disable, tabId);
+        yield call(disable, tab);
```

Comment:
Is that change voluntary ?

Reply:
Completely, see 1bc41efbb74f45d1425401b29f80352ed1209525